backend/accounts.go

```diff
@@ -244,3 +244,32 @@ func (backend *Backend) CreateAndPersistAccountConfig(
 			coinCode, nextAccountNumber, name, keystore, nil, accountsConfig)
 	})
 }
+
+func setTokenActive(accountCode string, tokenCode string, active bool, accountsConfig *config.AccountsConfig) error {
```

Comment:
Why this choice of a separate function in accounts.go? Wouldn't it be logical to make this a method of `config.AccountsConfig`? Then your last argument `accountsConfig` becomes the receiver.

Reply:
done